src/views/RequiredSignoffs/ViewSignoff/index.jsx

```diff
@@ -165,8 +178,37 @@ function ViewSignoff({ isNewSignoff, ...props }) {
     }
   };
 
-  // TODO: Add delete logic
-  const handleSignoffDelete = () => {};
+  const handleSignoffDelete = () =>
+    setDialogState({
+      ...dialogState,
+      open: true,
+      title: 'Delete Required Signoffs?',
```

Comment:
Should `Signoffs` be `Signoff`?

Reply:
Personally, I don't think so. We're deleting the entire collection of signoff requirements (that is to say, all of the roles that are currently required), so I think the plural is appropriate.